src/api/methods/getTransactionReceipt.js

```diff
@@ -17,18 +18,32 @@ module.exports = async (db, hash) => {
 
   const prevTx = await getPrevTx(db, tx);
 
+  const blockNumber = `0x${height.toString(16)}`;
+
+  const txLogs = (logs || []).map((log, i) => ({
+    transactionLogIndex: 0,
+    transactionIndex: txPos,
+    blockNumber,
+    transactionHash: hash,
+    address: bufferToHex(Buffer.from(log[0])),
```

Comment:
this will probably contain the sigHash of the transaction, not the address of the contract 🤔

Reply:
created an issue here: https://github.com/leapdao/leap-node/issues/331